README.md

```diff
@@ -27,10 +27,17 @@ conda activate movement-env
 
 ## Overview
 
-Pose estimation tools, such as [DeepLabCut](https://www.mackenziemathislab.org/deeplabcut) and [SLEAP](https://sleap.ai/) are now commonplace when processing video data of animal behaviour. There is not yet a standardised, easy-to-use way to process the pose tracks produced from these software packages.
-
-movement aims to provide a consistent modular interface to analyse pose tracks, allowing steps such as data cleaning, visualisation and motion quantification.
-We aim to support a range of pose estimation packages, along with 2D or 3D tracking of single or multiple individuals.
+Machine learning-based tools such as
+[DeepLabCut](https://www.mackenziemathislab.org/deeplabcut) and
+[SLEAP](https://sleap.ai/) have become commonplace for tracking the
+movements of animals and their body parts in videos.
```

Comment:
I like this way of framing it, i.e. the historical perspective and name-checking the fields. I'd just not mention "for pose estimation" (to avoid narrowing down things in the very first sentence), and perhaps do away with "In recent years", because that's bound to become dated.

So my proposed edit:

> Deep learning methods for motion tracking have revolutionised a range of scientific disciplines,
> from neuroscience and biomechanics, to conservation and ethology.
> Tools such as [DeepLabCut](https://www.mackenziemathislab.org/deeplabcut) or [SLEAP](https://sleap.ai/) now allow researchers to track animal movements
> in videos with remarkable accuracy, without requiring physical markers.

Reply:
Alright, the updated project overview (appearing in README and homepage) looks like this:

> Deep learning methods for motion tracking have revolutionised a range of
> scientific disciplines, from neuroscience and biomechanics, to conservation
> and ethology. Tools such as [DeepLabCut](https://www.mackenziemathislab.org/deeplabcut) and [SLEAP](https://sleap.ai/) 
> now allow researchers to track animal movements
> in videos with remarkable accuracy, without requiring physical markers.
> However, there is still a need for standardised, easy-to-use methods
> to process the tracks generated by these tools.
> 
> `movement` aims to provide a consistent, modular interface for analysing
> motion tracks, enabling steps such as data cleaning, visualisation,
> and motion quantification. We aim to support all popular animal tracking
> frameworks and file formats.
> 
> Find out more on our [mission and scope](https://movement.neuroinformatics.dev/community/mission-scope.html) statement and our [roadmap](https://movement.neuroinformatics.dev/community/roadmaps.html).